src/transactions/TransactionFrameBase.h

```diff
@@ -27,11 +27,12 @@ class TransactionFrameBase
                             TransactionEnvelope const& env);
 
     virtual bool apply(Application& app, AbstractLedgerTxn& ltx,
-                       TransactionMeta& meta) = 0;
+                       TransactionMeta& meta, TransactionResult& txResult) = 0;
```

Comment:
all methods exposed here should now be `const` (this would allow you to easily find remaining bugs related to mutating the object/caching)

Reply:
If there are any non const methods left in derived classes, it probably means they belong to `TestTransactionFrame` (or there is something we missed outside of "result", in which case let me know!)